src/transformers/generation/configuration_utils.py

```diff
@@ -218,8 +218,7 @@ class GenerationConfig(PushToHubMixin):
             Switch to sequential beam search and sequential topk for contrastive search to reduce peak memory.
             Used with beam search and contrastive search.
 
-
-        > Parameters that define the output variables of `generate`
+        > Parameters that define the output variables of generate
```

Comment:
Special formatting with ` does not work here:
<img width="457" alt="Screenshot 2024-05-02 at 12 24 48" src="https://github.com/huggingface/transformers/assets/12240844/a980e90f-ba0e-42b4-8481-ec4b31fde4b2">